externs/olx.js

```diff
@@ -5690,6 +5691,14 @@ olx.source.WMTSOptions.prototype.urls;
 
 
 /**
+ * Whether to wrap the world horizontally. Default is `false`.
```

Comment:
WMTS almost always uses a custom tile grid. If that is not a global one and users don't specify widths for each resolution, things will go wrong. When generating a WMTS layer from capabilities, the default does not matter anyway because wrapX will be correctly set.